app/views/tools/index.html.haml

```diff
@@ -0,0 +1,26 @@
+.grid-row
+  .column-two-thirds
+    %h1.heading-xlarge
+      Tools
+    
+    %p 
+      These tools will help you complete your submission.
+    
+    %h2.heading-medium
+      Unique Reference Number (URN) Lookup
+    
+    %p
+      You will be asked to supply a unique reference number for your customers in your template files, if you do not know your customer URN you can look it up in this Excel file.
```

Comment:
minor point, but I like to wrap long paragraphs like this in templates as it makes it easier to read the content in an editor window without impacting the semantics of the markup. I usually wrap after 80 characters or so.